server/channels/api4/system_test.go

```diff
@@ -83,12 +83,23 @@ func TestGetPing(t *testing.T) {
 		require.Contains(t, respString, "testvalue")
 	}, "ping feature flag test")
 
+	t.Run("ping root_status test", func(t *testing.T) {
```

Comment:
It would be great if the test also confirms that non-system admins don't see this root_status flag.

Reply:
Added another test with Client `th.Client`. I hope this works.